### PR TITLE
typsos in security doc (mostly possessive nouns)

### PR DESCRIPTION
**Pull request by combinatorist**

### CODE REVIEW

File: security.md [hunk 1]
```diff
@@ -444,13 +444,13 @@ new member. The key exchange is performed as follows:
...
 
--   In practice, the members account id and email and the organization id are
+-   In practice, the member's account id and email and the organization id are
     included in the respective signatures to protect these from tempering as
     well.
 
 -   Since `p` needs to be sufficiently short to be conveniently entered by
     hand, it can potentially be guessed by eavesdroppers which would allow them
-    to successfully perform a man-in-the-middle attach by injecting their own
+    to successfully perform a man-in-the-middle attack by injecting their own
```
Comment (combinatorist) [inline security.md:453]: wrong word

File: security.md [hunk 2]
```diff
@@ -745,7 +745,7 @@ order to eavesdrop on the connection and/or temper with messages in transit.
 
 MITM attacks may be launched in a multitude of ways, and even with technlogies
 like TLS, it is very hard to completely rule out that other parties may be
-listening in on the connection or even trying temper with the messages
+listening in on the connection or even trying to tamper with the messages
```
Comment (combinatorist) [inline security.md:748]: wrong word

File: security.md [hunk 3]
```diff
@@ -754,8 +754,8 @@ compromise the security of the application in any other way.
 -   Communication between the Padloc client and server is always secured through
     [Transport Layer Security](https://en.wikipedia.org/wiki/Transport_Layer_Security).
 -   No sensitive information is ever transmitted in plain text.
--   Transmitted data is protected from tempering through Padlocs strong [Authentication Mechanism](#authentication-and-data-transfer).
--   Padlocs [Key Exchange Mechanism](#trustless-server-mediated-key-exchange) is designed to be secure even over an untrusted connection.
+-   Transmitted data is protected from tampering through Padloc's strong [Authentication Mechanism](#authentication-and-data-transfer).
```
Comment (combinatorist) [inline security.md:757]: wrong word

File: security.md [hunk 4]
```diff
@@ -796,7 +796,7 @@ a very minimum, too costly to be worthwhile.
 
 [[TODO]]
 
-### Password spraying
+### Password Spraying
```
Comment (combinatorist) [inline security.md:799]: match casing of other headings

**Comment by MaKleSoft** [comment]
Thanks!

**Pull request merged**